cmd/vulcan-retirejs/main.go

```diff
@@ -53,17 +61,18 @@ func main() {
 			return checkstate.ErrAssetUnreachable
 		}
 
-		return scanTarget(ctx, target, logger, state, nil)
+		return scanTarget(ctx, target, assetType, logger, state, nil)
 	}
 	c := check.NewCheckFromHandler(checkName, run)
 	c.RunAndServe()
 
 }
 
-func scanTarget(ctx context.Context, target string, logger *logrus.Entry, state checkstate.State, args []string) error {
-	target, err := resolveTarget(target)
+func scanTarget(ctx context.Context, target, assetType string, logger *logrus.Entry, state checkstate.State, args []string) error {
+	target, err := resolveTarget(target, assetType)
 	if err != nil {
 		// Don't fail the check if the target can not be accessed.
+		// It shouldn't reach here due to previous check with helpers.IsReachable().
```

Comment:
This is not true for the case of a `Hostname`, because it could be reachable but not to expose a web server on ports 80 or 443.

Reply:
You are right. Removed the misleading comment here [2665a77](https://github.com/adevinta/vulcan-checks/pull/145/commits/2665a7703f3863e2b1a350b0335ce3e64e0fb048)